nestkernel/parameter.h

```diff
@@ -212,6 +214,7 @@ class ConstantParameter : public Parameter
     : Parameter( d )
   {
     value_ = getValue< double >( d, "value" );
+    parameter_returns_int_only_ = value_is_integer_( value_ );
```

Comment:
Yes, but it refers to the values inside the dictionary. I agree that it can be confusing, so I've changed the documentation for this, and for the similar places further down.